referral-form-data-process/package.json

```diff
@@ -4,15 +4,20 @@
   "license": "MIT",
   "scripts": {
     "lint": "prettier --check .",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "test": "jest"
   },
   "devDependencies": {
     "@types/aws-lambda": "^8.10.84",
     "@types/node": "^16.10.9",
     "prettier": "^2.4.1",
-    "typescript": "^4.4.4"
+    "typescript": "^4.4.4",
+    "@types/jest": "^26.0.24",
```

Comment:
Are you ok to update jest types to match the version of jest installed? Both jest types and jest should be on version 27.